pkg/client/conditions.go

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Not sure if this is the right place for clientset conditions @krousey @caesarxuchao @smarterclayton